playbook/buildfarmer/known_issues.md

```diff
@@ -49,6 +50,16 @@ To untrack a known issue, you may follow the steps below:
 2. Set the issue status to `CLOSED` in the buildfarmer database using [`issue_close.sql`](./buildfarmer_triage_tools.md#issue_closesql).
   * This script will set the issue status to `CLOSED` in the database (e.g., `./sql_run.sh issue_close.sql "https://github.com/my/issue/123"`).
 
+# Refresh Known Issues in the Database
+
+All the known issues are added manually to the database. They are kept open until someone closes them manually. To keep the database updated, we need to refresh the known issues periodically.
+
+To refresh the known issues, you may follow the steps below:
+1. Run [`./refresh_known_open_issues.sh`](./buildfarmer_triage_tools.md#refresh_known_open_issuessh)
```

Comment:
Do we have anywhere docs about setting up the tools to make this script runnable?

Reply:
I don't think so. I will add some information about it

Reply:
Added note in: 6b93297